ios/FluentUI/BarButtonItems/BarButtonItems.swift

```diff
@@ -19,6 +19,16 @@ public final class BarButtonItems: NSObject {
         return button
     }
 
+    /// When adding this barButtonItem to the view, tint it with appropriate app color UIColor(light: Colors.primary(for: window), dark: Colors.textDominant)
```

Comment:
we shouldn't  add this here which increase the library code side. please move it to demo app code to illustrate different types of barbuttonitem

Reply:
Restored original BatButtonItems class and moved this code in Demo project

Reply:
Moved it to Demo project